Tests/js/views/dashboard/assets/ez-dashboardblockallcontentview-tests.js

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockallcontentview-tests', function (Y) {
+    'use strict';
+
+    var renderTest,
+        getDataTest,
+        uiEventsTest,
+        CLASS_LOADING = 'is-loading',
+        CLASS_ROW_SELECTED = 'is-row-selected',
+        SELECTOR_CONTENT = '.ez-allcontent-block-content',
+        SELECTOR_ROW = '.ez-allcontent-block-row',
+        SELECTOR_OUTSIDE = '.outside',
+        VIEW_CONFIG = {container: '.container'};
+
+    renderTest = new Y.Test.Case({
+        name: 'eZ Dashboard All Content Block View render test',
+
+        setUp: function () {
+            this.view = new Y.eZ.DashboardBlockAllContentView(VIEW_CONFIG);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        'Should render view correctly with loading state': function () {
+            var view = this.view,
+                templateCalled = false,
+                origTpl = view.template,
+                container = view.get('container');
+
+            view.template = function () {
+                templateCalled = true;
+
+                return origTpl.apply(this, arguments);
+            };
+
+            view.render();
+
+            Y.Assert.isTrue(templateCalled, 'The template should have been used to render view');
+            Y.Assert.areSame(0, container.one(SELECTOR_CONTENT).all('tr').size(), 'Should not render any row');
+            Y.Assert.isTrue(container.hasClass(CLASS_LOADING), 'Should add the loading state CSS class to the view container');
+        },
+
+        'Should render view correctly with content inside': function () {
```

Comment:
same as in the previous test, should be something like 'Should pass the content item structs to the template'